qpylib/qpylib.py

```diff
@@ -97,17 +97,17 @@ def get_console_fqdn():
 
 # pylint: disable=invalid-name, too-many-arguments
 def REST(rest_action, request_url, version=None, headers=None, data=None,
-         params=None, json_body=None, verify=None, timeout=60, **kwargs):
+         params=None, json_body=None, verify=True, timeout=60, **kwargs):
     ''' Invokes a rest_action request to request_url using the Python requests module.
         Returns a requests.Response object.
         Raises ValueError if rest_action is not one of GET, PUT, POST, DELETE.
     '''
     if util_qpylib.is_sdk():
-        rest_func = rest_qpylib.sdk_rest
-    else:
-        rest_func = rest_qpylib.live_rest
-    return rest_func(rest_action, request_url, version, headers, data,
-                     params, json_body, verify, timeout, **kwargs)
+        # To be completed. Default to no verification for now.
```

Comment:
Having 2 returns is a bit clunky. Instead can we have a variable called `rest_verify`, say, and pass that in a single return.

`rest_verify = False if is_sdk() else verify`

Reply:
sounds good to me. Updated it to the above ;)